docs/ai/includes/vector-databases.md

```diff
@@ -22,3 +24,5 @@ AI applications often use data vector databases and services to improve relevanc
 - [Redis](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Redis)
 - [Sqlite](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Sqlite)
 - [Weaviate](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.Weaviate)
```

Comment:
We should still promote 3rd party SDKs for vector DBs, which can be used without SK. Maintaining these from the prior page. While we are promoting SK, we still want to show there's an ecosystem outside of it.

| [Azure.Search.Documents](https://www.nuget.org/packages/Azure.Search.Documents/) | Azure AI Search | [Azure SDK for .NET](https://github.com/Azure/azure-sdk-for-net) (Microsoft) | [Azure AI Search client library for .NET](/dotnet/api/overview/azure/search.documents-readme) |
| [Milvus.Client](https://www.nuget.org/packages/Milvus.Client) | Milvus Vector Database | [Milvus](https://milvus.io/) | [Install Milvus C# SDK](https://milvus.io/docs/v2.2.x/install-csharp.md) |
| [Qdrant.Client](https://www.nuget.org/packages/Qdrant.Client) | Qdrant Vector Database | [Qdrant](https://qdrant.tech) | [Qdrant .NET SDK](https://github.com/qdrant/qdrant-dotnet) |